bin/commands/scripts/update_translations.js

```diff
@@ -196,26 +197,31 @@ function getFile(resource, language) {
     });
 }
 
+if (!manager.getConfig("api").offline_mode) {
```

Comment:
Can you recheck if this works correctly when changing setting in UI and running command line?